<launch>

    <node pkg="lasr_object_detection_yolo" name="yolo_object_detection_server" type="yolo_object_detection_server.py"/>
    <node pkg="lasr_perception_server" name="perception_server" type="perception_server.py"/>

</launch>